load_tests/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.7
+
+EXPOSE 8089
+WORKDIR /app
+
+# install poetry
+
+ARG POETRY_VERSION="1.0.0"
+RUN \
+  pip install --upgrade pip \
+  && pip install "poetry==${POETRY_VERSION}" \
+  && poetry config virtualenvs.create false
+
+COPY load_tests .
+COPY integration-tests-en integration-tests-en
+RUN poetry install -vvv --no-dev
+
+ENV INTEGRATION_TEST_PATH "integration-tests-en"
+
+ENTRYPOINT ["./run.sh", "socketio"]
+CMD ["--no-web", "-c", "5", "-r", "5", "--run-time", "1200s"]
```

Comment:
What are those parameters? Should we define the values with env variables instead to allow overriding them?

Reply:
It's how they do it in the dexter project, I followed their convention in case it's necessary for their plans to automate load tests.

Reply:
All right let's keep it that way then